tron/bin/check_tron_dynamodb_staleness.py

```diff
@@ -0,0 +1,110 @@
+    '''
+    Get all sorted timestamps, and only count the actions that actually ran
+    '''
+    timestamps = []
+    job_runs = job['runs']
+    for run in job_runs:
+        for action in run['runs']:
+            if action.get('start_time') and action.get('state') != 'scheduled':
+                timestamps.append(action.get('start_time'))
+    timestamps.sort()
+    return timestamps[-1] if timestamps else None
+
+
+def parse_cli():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "-w",
+        "--working-dir",
+        default=DEFAULT_WORKING_DIR,
+        help="Working directory for the Tron daemon, default %(default)s",
+    )
+    parser.add_argument(
+        "-c",
+        "--config-path",
+        default=DEFAULT_CONF_PATH,
+        help="File path to the Tron configuration file",
+    )
+    parser.add_argument(
+        "--job-for-stateless-alert",
+        required=True,
+        help="The job name to read timestamp from",
+    )
+    parser.add_argument(
+        "--staleness-threshold",
+        default=DEFAULT_STALENESS_THRESHOLD,
+        help="how long (in mins) to wait to alert after the last timestamp of the job",
+    )
+    args = parser.parse_args()
+    args.working_dir = os.path.abspath(args.working_dir)
+    args.config_path = os.path.join(
+        args.working_dir,
+        args.config_path,
+    )
+    return args
+
+
+def read_config(args):
+    return manager.ConfigManager(args.config_path).load().get_master().state_persistence
+
+
+def main():
+    # Fetch configs. You can find the arguments in puppet.
+    args = parse_cli()
+    persistence_config = read_config(args)
+    store_type = schema.StatePersistenceTypes(persistence_config.store_type)
+    job_for_stateless_alert = args.job_for_stateless_alert
+
+    # Alert for DynamoDB
+    if store_type == schema.StatePersistenceTypes.dynamodb:
+        # Fetch job state from dynamodb
+        dynamodb_region = persistence_config.dynamodb_region
+        table_name = persistence_config.table_name
+        store = DynamoDBStateStore(table_name, dynamodb_region)
+        key = store.build_key('job_state', '{}'.format(job_for_stateless_alert))
+        try:
+            job = store.restore([key])[key]
+        except Exception as e:
+            logging.error(f'Failed to retreive status for job {job_for_stateless_alert} due to {str(e)}')
+            sys.exit(1)
+
+        # Exit if the job never runs.
+        last_run_time = get_last_run_time(job)
+        if not last_run_time:
+            logging.error(f'No last run for {key} found. If the job was just added, it might take some time for it to run')
+            sys.exit(1)
+
+        # Alert if timestamp is not updated after staleness_threshold
+        stateless_for_secs = time.time() - last_run_time.timestamp()
+        if stateless_for_secs > args.staleness_threshold:
+            logging.error(f'DynamoDB has not been updated for {stateless_for_secs} mins')
```

Comment:
can you include the job name key here too, to remind the responder exactly what it was looking for?